### DISCUSSION TITLE
Generating a html report

### DISCUSSION THREAD

**Discussion by MathieuSoysal** [sourcefrog/cargo-mutants#281] [Ideas]
First thank you for your awesome project !

I'm wondered what do you think about generating a html report ? But I don't know if have a plugin to do that ?

Example of html report for mutation testing: https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/robo-coasters-example/master#mutant

**Comment by sourcefrog** [comment]
Yep, that would be nice! It would be good to eventually have this built in but if you wish to prototype it externally then all the data should be in `mutants.out/outcomes.json`.

**Comment by max-te** [comment]
I came across https://github.com/stryker-mutator/mutation-testing-elements and put together a [protoype](https://github.com/max-te/mutant-report) based on that. Here is an example report for the url crate: https://max-te.github.io/mutant-report/url/#mutant

**Reply by sourcefrog** [reply]
Wow that looks great!
